pkg/client/openstack/admin/client.go

```diff
@@ -251,3 +260,54 @@ func (c *adminClient) getRoleID(roleName string) (string, error) {
 	return "", fmt.Errorf("Role %s not found", roleName)
 
 }
+
+func (c *adminClient) CreateContainer(containerName, serviceUserName, serviceUserDomainName, serviceUserProjectID string) error {
```

Comment:
the `projectID` is not tied to the serviceUser we want to grant access. I fact the user is from another domain even. So I would name it differently.
```suggestion
func (c *adminClient) CreateContainer(projectID string, containerName, serviceUsername, serviceUserDomainName) error {
```